web_src/less/_review.less

```diff
@@ -317,3 +317,11 @@ a.blob-excerpt:hover {
   width: 72px;
   height: 10px;
 }
+
+.diff-file-box {
+  border-radius: .285rem; // Just like ui.top.attached.header
+}
+
+.diff-file-box:target {
+  box-shadow: 0 0 0 3px var(--color-accent);
+}
```

Comment:
There is already a `.diff-file-box` in 

https://github.com/go-gitea/gitea/blob/be93e48ccb73acfaa372e56217105d3420e1a40e/web_src/less/_repository.less#L1706

And the `border-radius` doesn't look necessary. On your screenshot, the bottom border corners are still right angles.

Reply:
1. `_repository.less` is a completely different file, and the selector you're speaking about is nested. While we _can_ use it, I'm not a fan of nested selectors as they make the outcome so difficult to predict. Additionally, `_review` should be loaded less often than `_repository` and only when needed. 
2. The top `border-radius` is needed as the `diff-file-header` sets the `border-radius: 0.285rem` (`ui.top.attached.header`). Otherwise, it looks like this: 
![no border-radius](https://user-images.githubusercontent.com/51889757/218303643-8d528452-2a2a-4108-9e9d-a6fdcfacb54f.png). What we could think about is moving the `border-radius` into `:target`, but why?
Why should it change its boundaries only when targeted?
3. I know that as well. However, I've wasted half an hour already on it and still haven't found any solution. The problem seems to be that `Fomantic UI` forcefully sets the `border-radius:0` for the table, and I haven't found **any** solution yet. At that point, I simply surrendered and I think the functional aspect of this enhancement is present. If anyone has any better ideas, please feel free to commit directly on this PR.